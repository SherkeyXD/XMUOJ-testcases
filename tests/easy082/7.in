280
8300 7952
6896 3939
3984 6600
2710 736
8960 8322
6428 6224
4388 7976
660 5575
8472 7820
2188 383
2594 1952
9840 3990
2380 8920
76 1324
1190 7398
7220 234
9827 841
1870 5530
4134 1392
4924 7272
8520 4290
1157 4952
8090 638
2588 3224
6784 7550
9593 8760
9568 4105
6059 4494
555 5200
7335 1516
500 6092
1340 4580
5592 6200
9378 1200
3420 5740
4279 4662
480 266
150 9830
732 8966
5550 8454
2900 7936
5490 3885
5178 1264
5063 4792
1667 8037
5802 9550
9700 9404
5770 1441
3241 6726
1290 3200
2936 7250
9572 1660
5590 3982
9677 5129
4757 1350
4688 6600
9436 7448
7050 8874
6216 5612
6890 6286
9446 1877
3236 3630
8434 2052
2378 6832
9968 5773
3979 1949
3027 1359
5147 5820
3474 3835
6610 4364
5645 3085
8951 7120
7313 8222
6501 3072
4786 7413
6960 5020
2209 1775
2520 8131
5081 6952
1514 540
8844 80
4175 8422
927 6318
4700 4854
7880 2400
7406 7467
5372 7580
4410 2760
9794 718
9991 2168
7312 6856
9186 5520
2265 8696
2340 1558
1201 5872
3786 4920
2850 2198
4247 2576
7024 1125
1131 5041
8214 5624
5352 5485
9036 1072
9150 5496
5809 3980
3109 9120
9076 304
4296 850
6645 7180
9341 1630
2162 9824
408 2445
9508 5712
8496 4424
4442 8762
3571 7958
5856 5140
7194 7104
3843 7767
6071 8276
8000 6000
4720 8598
9424 8744
857 2704
3310 4250
309 933
2235 502
1746 8730
7600 4215
3314 2334
8277 4816
3700 5425
8000 4017
190 800
3742 1129
1572 536
5574 4042
1345 8950
1288 3276
174 9648
5524 9520
9715 6
776 6425
876 8106
8648 7086
5837 270
521 8640
3290 1250
2392 9225
1516 7100
8385 3920
6400 3028
5068 775
4688 5380
232 5360
4060 774
7048 6796
7202 8256
8660 4080
7414 2770
5962 6420
8016 3197
6860 6160
7676 8868
1124 2066
2680 8537
663 2812
7320 7948
7240 855
4238 5914
9172 8192
1488 2400
6944 5074
3390 1329
9575 2720
4376 3297
1540 7245
4171 6261
500 4915
9401 9820
4032 2128
642 402
8664 8992
3277 6643
4592 9925
8176 4238
2808 2906
1461 8173
4380 7128
6360 315
3625 6271
7640 4362
2292 4360
3833 2703
1128 6275
9633 5720
9215 5200
6587 3526
5000 404
5256 460
4575 2704
5048 1802
7308 5479
486 1600
7351 3280
3270 2090
1088 2150
9450 5588
8690 6800
6132 7730
3432 2022
2715 7680
6684 4883
2200 4834
6722 4293
8900 7331
8300 8800
3588 8361
4808 1696
7898 9660
6632 7980
96 4670
2680 50
5928 9930
7024 1848
3922 4975
300 5685
7314 8387
6900 4540
465 8989
6323 2991
9048 7900
3100 5527
616 1750
1810 7975
2080 8857
8788 2168
3500 2372
5247 2752
4400 7613
5600 4110
5130 2526
6310 7959
604 8108
2900 7890
9478 1190
7100 5380
3120 5565
500 6234
2558 1385
5408 9400
9940 6176
4160 1378
4890 7620
1744 3896
916 8632
4780 8225
7848 7073
3748 2416
7078 8040
3770 2096
3296 4856
1060 1519
4092 2803
6352 9632
1350 1772
6182 5793
2087 9636
3220 5734
3480 384
1990 2744
4132 1866
225 9487
4135 9747
432 5200
9380 5579
3882 2768
4968 6998
851 1998
7725 4970
