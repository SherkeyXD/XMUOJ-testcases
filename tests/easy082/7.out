0
56
376
0
0
256
616
0
776
72
336
0
0
176
0
0
627
0
56
776
0
601
0
336
376
1
568
961
625
625
0
0
376
376
0
41
0
0
224
0
0
0
56
881
427
624
0
0
241
0
376
576
0
237
249
376
56
0
56
0
416
176
856
216
768
619
763
401
624
0
125
1
169
1
256
0
249
0
561
776
776
625
569
0
0
536
976
0
16
121
416
576
625
0
401
576
0
921
624
331
816
632
496
0
201
801
176
376
625
201
336
368
736
656
64
761
576
496
507
321
0
0
176
601
0
429
625
976
0
16
361
0
0
0
472
96
976
625
416
976
376
625
376
376
464
449
801
0
432
376
625
0
432
576
376
0
936
536
0
576
176
56
0
776
376
0
81
0
0
864
216
376
96
0
625
376
0
371
0
1
776
664
296
133
432
776
544
381
0
0
625
0
176
537
432
601
625
209
0
176
625
304
272
376
1
0
624
0
0
624
624
625
304
0
552
0
0
288
656
976
376
776
0
24
176
568
0
304
0
625
227
376
0
376
0
0
896
0
241
0
0
0
0
736
0
824
0
0
0
968
376
0
0
0
856
256
0
408
536
176
0
896
0
688
896
0
432
441
0
0
0
24
625
375
376
0
176
224
801
625
